nginx/conf.d/scalelite-ssl.template

```diff
@@ -4,7 +4,9 @@ server {
 	listen 80 default_server;
 	listen [::]:80 default_server;
 
-	include conf.d/scalelite.common;
+	location / {
+    return  301 https://$host$request_uri;
+  }
```

Comment:
Some indentation issues here, try to be consistent with the rest of the file.

Reply:
The original file uses real tab, not space.

Reply:
Well, we should replace tabs by spaces. Let the editors do their job :)